### PR TITLE
docs: troubleshooting Gitpod ports

### PR DESCRIPTION
**Pull request by zairahira**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

This should be useful when the service is not coming up on `port:8000`.

### CODE REVIEW

File: docs/how-to-setup-freecodecamp-locally.md [hunk 1]
```diff
@@ -39,6 +39,18 @@ There are various ways to launch an GitPod workspace:
...
+
+Sometimes the service on `port:8000` doesn't go live. This is common when you are restarting an inactive workspace.
+
+If the service is not coming up on `port:8000`, you can troubleshoot using these steps:
+
+- **Start the server**: Run `pnpm run develop:server` in one terminal window from the root project directory (`/workspace/freeCodeCamp`) to start the server.
+
+- **Start the client**: In another terminal window, run `pnpm run develop -- -H '0.0.0.0'` from the client directory (`/workspace/freeCodeCamp/client`) to start the client.
+
+This should make the `port:8000` avaialable and you can preview the rendered pages in a new tab.
```
Comment (ShaunSHamilton) [inline docs/how-to-setup-freecodecamp-locally.md:52]:
```suggestion
This should make port `8000` available.
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Thank you for taking the time to document this.

Although, I am not sure about the need here? I have never come across this, and I daily use Gitpod.

**Comment by zairahira** [comment]
I usually come across this issue, so I though it would be good to add it for reference.

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍

**Review by Sembauke — APPROVED** [review]

**Pull request merged**